src/mon/NVMeofGwTypes.h

```diff
@@ -228,13 +229,6 @@ struct NvmeGwTimerState {
   NvmeGwTimerState() {};
 };
 
-typedef struct Gw_Epoch {
-    epoch_t epoch;
-    Gw_Epoch(epoch_t epoch) : epoch(epoch){
-    };
-    Gw_Epoch():Gw_Epoch(0) {};
-}GwEpoch;
-
```

Comment:
Why is this introduced introduced in the prior commit and then removed in this one?  Fix the prior commit to not introduce it in the first place.